### ISSUE TITLE
Change dependencies view to a day range, not a time range

### ISSUE THREAD

**Issue by codefromthecrypt** [openzipkin/zipkin#1185]
The spark job only works on daily buckets, and that's likely to be the way out for all storage services. It would be more clear if we chose day ranges as opposed to time ranges, I think.

cc @eirslett @virtuald @jcarres-mdsol @yurishkuro @travisby @anuraaga

**Comment by codefromthecrypt** [comment]
by "only works on daily buckets" I mean that in cassandra and also elasticsearch, we store links in daily buckets. when we get to pre-aggregating mysql, it would be simplest to follow that pattern

**Comment by adriancole** [comment]
ps I think the best would be to set the default day to yesterday. This will help ensure data is available

**Comment by adriancole** [comment]
yesterday is semantically equivalent to today - 1 DAY lookback

**Comment by jcarres-mdsol** [comment]
Good idea, I do see much value in using time-ranges smaller than one day
